youtube_dl/extractor/tvn24.py

```diff
@@ -39,7 +39,10 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
-        title = self._og_search_title(webpage)
+        title = self._html_search_regex(
+            r'<span[^>]+class="standardHeader1[^"]*"[^>]*>\s*(.+?)\s*</span>',
```

Comment:
1. It's not a title. You must extract title of the video not series.
2. `class=standardHeader1` pattern is too ambiguous and not future-proof since it occurs multiple times on the page though not in span currently.